lib/commands/create.js

```diff
@@ -11,21 +11,26 @@ module.exports = function (program) {
   program
     .command('create <shortname-without-spaces>')
     .description(stripIndent`
-      Create an Apostrophe project (using the A3 boilerplate by default).
+      Create an Apostrophe project (using the A3 Essentials starter kit by default).
       Example: \`apos create my-website\`
     `)
     .option('--a2', 'Use the Apostrophe 2 boilerplate')
-    .option('--starter <url>', 'Use a specific git repository to use as the project starter')
+    .option('--starter <url>', 'Use a specific git repository to use as the project starter.\n You can also use the short name of any Apostrophe starter kit, e.g. "ecommerce"')
     .action(async function(shortName, options) {
-      if (!options.starter && options.boilerplate) {
-        // Support the old boilerplate flag to be nice.
-        options.starter = options.boilerplate;
+      // If options.starter is undefined, use config.A3_BOILERPLATE
+      const input = options.starter ? options.starter : config.A3_BOILERPLATE;
+
+      // Check for complete repo URL, starterkit name, or incomplete URL fallback
+      let boilerplateUrl, newUrl;
+      if (/^\w+:/i.test(input)) {
+        boilerplateUrl = newUrl = input;
+      } else if (input.includes('/')) {
+        boilerplateUrl = newUrl = `https://github.com/${input.startsWith('/') ? input.slice(1) : input}`;
+      } else {
+        boilerplateUrl = `https://github.com/apostrophecms/starter-kit-${input}.git`;
+        newUrl = `https://github.com/@apostrophecms/${shortName}`;
```

Comment:
What exactly is `newUrl` for?

This URL isn't right for sure — it has a `@`, which is an npm thing.

But beyond that I'm not sure what the goal of `newUrl` is.